app/js/layout/App.jsx

```diff
@@ -66,35 +68,41 @@ class App extends Component {
-            {path: '/buy/contactForm', component: BuyerContactContainer},
-            {path: '/buy/offer', component: OfferTradeContainer}
-          ]}/>
-
-          <Wizard path="/sell/" steps={[
-            {path: '/sell/asset', component: SellerAssetContainer},
-            {path: '/sell/location', component: SellerLocationContainer},
-            {path: '/sell/payment-methods', component: SellerPaymentMethodsContainer},
-            {path: '/sell/currency', component: SellerCurrencyContainer},
-            {path: '/sell/margin', component: SellerMarginContainer, nextLabel: 'Confirm price'},
-            {path: '/sell/contact', component: SellerContactContainer, nextLabel: 'Post the offer'}
-          ]}/>
-
-          <Route path="/tmp/price" component={PriceContainer}/>
-          <Route path="/tmp/escrows" component={EscrowsContainer}/>
-          <Route path="/tmp/map" component={MapContainer}/>
-          <Route path="/tmp/signature" component={SignatureContainer}/>
-          <Route path="/tmp/arbitration" component={ArbitrationContainer}/>
+          <Switch>
+            <Route exact path="/" component={HomeContainer}/>
+            <Route exact path="/profile" component={ProfileContainer}/>
+            <Route exact path="/profile/edit" component={EditProfileContainer}/>
+            <Route exact path="/license" component={LicenseContainer}/>
+
+
+            <Route exact path="/buy" component={OfferListContainer}/>
+            <Route exact path="/buy/map" component={MapContainer}/>
+            <Route exact path="/buy/list" component={BankOfferListContainer}/>
+            <Route exact path="/buy/profile/:address" component={SellerProfileContainer}/>
+            <Wizard path="/buy/contact" steps={[
+              {path: '/buy/contactForm', component: BuyerContactContainer},
+              {path: '/buy/offer', component: OfferTradeContainer}
+            ]}/>
+
+            {this.props.isLicenseOwner &&
+              <Wizard path="/sell/" steps={[
+                {path: '/sell/asset', component: SellerAssetContainer},
+                {path: '/sell/location', component: SellerLocationContainer},
+                {path: '/sell/payment-methods', component: SellerPaymentMethodsContainer},
+                {path: '/sell/currency', component: SellerCurrencyContainer},
+                {path: '/sell/margin', component: SellerMarginContainer, nextLabel: 'Confirm price'},
+                {path: '/sell/contact', component: SellerContactContainer, nextLabel: 'Post the offer'}
+              ]}/>
+            }
+
+            <Route path="/tmp/price" component={PriceContainer}/>
+            <Route path="/tmp/escrows" component={EscrowsContainer}/>
+            <Route path="/tmp/map" component={MapContainer}/>
+            <Route path="/tmp/signature" component={SignatureContainer}/>
+            <Route path="/tmp/arbitration" component={ArbitrationContainer}/>
+
+            <Redirect to="/"/>
```

Comment:
What does this do? Asking for my knowledge.

Reply:
This is a catch all in case the route is not known, redirect to home
We can also add a 404 page once we/if have it

Reply:
Ok, thanks for the answer. Indeed, a 404 page would be better. I'll add a Pivotal task